app/helpers/application_helper.rb

```diff
@@ -322,4 +322,19 @@ def creator_settings_form?
     creator = User.with_role(:creator).first
     !creator.checked_code_of_conduct && !creator.checked_terms_and_conditions
   end
+
+  # A wrapper for the `inline_svg_tag` helper specifically for Crayons icons.
+  #
+  # @param name [String] the icon name. The ".svg" file extension will be added
+  #   automatically if missing.
+  # @param css_class [String] additional CSS classes, "crayons-icon" is always
+  #   included.
+  # @return [String] the SVG tag.
+  def crayons_icon_tag(name, css_class: nil, **opts)
+    icon_name = name.ends_with?(".svg") ? name : "#{name}.svg"
+    icon_class = "crayons-icon"
+    icon_class = "#{icon_class} #{css_class}" if css_class
+
+    inline_svg_tag(icon_name, aria: true, width: 24, height: 24, class: icon_class, **opts)
```

Comment:
I've included `aria: true` by default since this adds `role="img"`. It also seems like we pretty much manually added this for all our existing icons anyway.